nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -1,14 +1,45 @@
 const express = require("express");
 const router = express.Router();
 
-const meals = require("./../data/meals.json");
+const meals = require("../data/meals.json");
 
 router.get("/", async (request, response) => {
-  try {
-    console.log(meals);
-    console.log("in /api/meals");
-  } catch (error) {
-    throw error;
+  console.log(meals);
+  console.log("in /api/meals");
+  const query = request.query
+  console.log(request.query)
+  if ((req.query.maxPrice && isNaN(parseInt(req.query.maxPrice))) ||
```

Comment:
When I call the endpoint, I get an error: 
`ReferenceError: req is not defined`
It is because in line 6 you were writing request, and here in line 11, you are trying to access 'req', which is not defined. Therefore, here you should change req to request.